.github/workflows/tests-misc.yml

```diff
@@ -0,0 +1,379 @@
+          cargo clean
+          # build again
+          time forklift cargo build -q --locked --release -p westend-runtime -p rococo-runtime
+          # confirm checksum
+          sha256sum -c checksum.sha256
+
+  cargo-check-benches-branches:
+    needs: [ set-image ]
+    if: ${{ github.event_name == 'pull_request' }}
+    timeout-minutes: 60
+    outputs:
+      branch: ${{ steps.branch.outputs.branch }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Branch
+        id: branch
+        run: |
+          echo "branch=['${{ github.base_ref }}', '${{ github.head_ref }}']" >> $GITHUB_OUTPUT
+
+  cargo-check-benches:
+    needs: [ set-image, cargo-check-benches-branches ]
+    timeout-minutes: 60
+    strategy:
+      matrix:
+        branch: ${{ fromJSON(needs.cargo-check-benches-branches.outputs.branch) }}
+    runs-on: arc-runners-polkadot-sdk-beefy
+    container:
+      image: ${{ needs.set-image.outputs.IMAGE }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ matrix.branch }}
+
+      - name: script
+        run: |
+          ARTIFACTS_DIR=./artifacts
+          BENCH_TRIE_READ=::trie::read::small
+          BENCH_NODE_IMPORT=::node::import::sr25519::transfer_keep_alive::paritydb::small
+          mkdir -p $ARTIFACTS_DIR
+
+          SKIP_WASM_BUILD=1 forklift time cargo check --locked --benches --all;
+          forklift cargo run --locked --release -p node-bench -- $BENCH_TRIE_READ --json | tee $ARTIFACTS_DIR/bench_trie_read_small.json;
+          forklift cargo run --locked --release -p node-bench -- $BENCH_NODE_IMPORT --json | tee $ARTIFACTS_DIR/bench_transfer_keep_alive.json
+
+      - name: Upload artifacts
+        uses: actions/upload-artifact@v4.3.6
+        with:
+          path: ./artifacts
+          name: cargo-check-benches-${{ matrix.branch }}-${{ github.sha }}
+          retention-days: 1
+
+  node-bench-regression-guard:
+    timeout-minutes: 20
+    runs-on: arc-runners-polkadot-sdk-default
+    needs: [ set-image, cargo-check-benches ]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4.1.7
+
+      - name: Extract branch name
+        shell: bash
+        run: echo "branch=${GITHUB_HEAD_REF:-${GITHUB_REF#refs/heads/}}" >> $GITHUB_OUTPUT
+        id: extract_branch
+
+      - name: Download artifact (master run)
+        uses: actions/download-artifact@v4.1.8
+        with:
+          name: cargo-check-benches-${{ github.base_ref }}-${{ github.sha }}
+          path: ./artifacts/master
+
+      - name: Download artifact (current run)
+        uses: actions/download-artifact@v4.1.8
+        with:
+          name: cargo-check-benches-${{ github.head_ref }}-${{ github.sha }}
+          path: ./artifacts/current
+
+      - name: script
+        run: |
+          echo "------- IMPORTANT -------"
+          echo "node-bench-regression-guard depends on the results of a cargo-check-benches job"
```

Comment:
It makes sense to put this in the GitHub output like [here](https://github.com/paritytech/polkadot-sdk/pull/5340/files#diff-ca6ddd9d3cdff8f696fe610915c9cd9316882794d8ed1e1e4dc6be392c0eb3e3R136)

Reply:
added

Reply:
now added:)